entries/p/pixieset.com.json

```diff
@@ -1,12 +1,12 @@
 {
-    "Pixieset": {
-        "domain": "pixieset.com",
-        "tfa": [
-            "totp"
-        ],
-        "documentation": "https://help.pixieset.com/hc/en-us/articles/360046803431",
-        "keywords": [
-            "other"
-        ]
-    }
-}
+	"Pixieset": {
+		"domain": "pixieset.com",
+		"tfa": [
+			"totp"
+		],
+		"documentation": "https://help.pixieset.com/hc/en-us/articles/360046803431",
+		"keywords": [
+			"other"
+		]
+	}
+}
```

Comment:
Like I said in another PR, this isn't something worth fixing IMO. As long as it is valid JSON and complies with the JSON schema it's fine as it is.

When you create a new JSON file or do major reworks of an existing JSON file you can do the correct space indentation. That way we get more time for things like PRs with site additions :wink:

Reply:
> Like I said in another PR, this isn't something worth fixing IMO. As long as it is valid JSON and complies with the JSON schema it's fine as it is.
> 
> When you create a new JSON file or do major reworks of an existing JSON file you can do the correct space indentation. That way we get more time for things like PRs with site additions 😉

OK